### PR TITLE
Feature: Adds new nonzero command

### PR DESCRIPTION
**Pull request by jose-donato**
* adds command to check the addresses with non-zero balance over time in a certain blockchain

image

___

Decided after a quick chat on [gamestonkterminal discord](https://discord.com/invite/Up2QGbMKHY) with @deeleeramone:

image

### CODE REVIEW

File: gamestonk_terminal/cryptocurrency/due_diligence/glassnode_model.py [hunk 1]
```diff
@@ -161,6 +161,49 @@
...
+    parameters = {
+        "api_key": cfg.API_GLASSNODE_KEY,
+        "a": asset,
+        "i": interval,
+        "s": str(since),
+        "u": str(until),
+    }
+
+    r = requests.get(url, params=parameters)
+    print(url, parameters)
```
Comment (DidierRLopes) [inline gamestonk_terminal/cryptocurrency/due_diligence/glassnode_model.py:198]: Screenshot 2021-12-30 at 02 24 39

random print in the wild!

**Review by DidierRLopes — CHANGES_REQUESTED** [review]
Just remove that line and is good to go!

File: gamestonk_terminal/cryptocurrency/onchain/onchain_controller.py [hunk 2]
```diff
@@ -77,7 +77,6 @@ class OnchainController:
         "th",
         "prices",
         "address",
-        "active",
```
Comment (DidierRLopes) [inline gamestonk_terminal/cryptocurrency/onchain/onchain_controller.py:80]: Was this on purpose?

**Comment by DidierRLopes** [comment]
Can we add this function in a .gst file?

File: gamestonk_terminal/cryptocurrency/onchain/onchain_controller.py (hunk 2 above)
Comment (jose-donato) [reply gamestonk_terminal/cryptocurrency/onchain/onchain_controller.py:80]: yea, when i moved active command from onchain to due dilligence menu i forgot to remove it from the commands list of onchain

**Comment by jose-donato** [comment]
> Just remove that line and is good to go!

🤦

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**